models/grading/grading_events.sql

```diff
@@ -18,10 +18,12 @@ select
     splitByString('/', course_id)[-1] as course_key,
     org,
     verb_id,
-    JSONExtractFloat(event, 'result', 'score', 'scaled') as scaled_score
+    coalesce(JSONExtractFloat(event, 'result', 'score', 'scaled'), 0) as scaled_score
```

Comment:
Nitpick: `JSONExtractFloat` will default to 0 if `result.score.scaled` doesn't exist, so I don't think this `coalesce` is necessary.